README.md

```diff
@@ -32,6 +30,17 @@ Obviously, if you're installing this onto a system that does not use
 OpenRC by default then you may wish to backup the above listed files,
 remove them and then install so that the OS hooks into OpenRC.
 
+If you are installing this onto a system which has separate / and /usr
+file systems and expects / to be mounted before /usr, you will need to
+make sure the appropriate pieces are installed on / by passing the
+appropriate options to "meson setup" as follows:
+
+```
+$ meson setup \
+        --bindir /bin --libdir /lib64 --libexecdir /lib --sbindir /sbin \
```

Comment:
all of those are basically meson's default, instead i think we should recommend:
```suggestion
        --prefix / --libexecdir lib \
```